Makefile

```diff
@@ -97,6 +97,9 @@ TEST_PATTERN ?= /
 run-hive: build-image setup-hive ## 🧪 Run Hive testing suite
 	cd hive && ./hive --sim $(SIMULATION) --client ethereumrust --sim.limit "$(TEST_PATTERN)"
 
+run-hive-on-latest: setup-hive ## 🧪 Run Hive testing suite with the latest docker image
```

Comment:
Why do we need this new make target? Looks identical to run-hive

Reply:
It is, the only difference is it doesn't build the image as a prereq for the task, which allows us to resuse the image we already built as part of the workflow.